Formula/boxes.rb

```diff
@@ -13,14 +13,20 @@ class Boxes < Formula
     sha256 mojave:        "7d2bf5e65b67fbacc15bbf366decb57486393b5f53732627f09c286557e7cb74"
   end
 
+  depends_on "bison"
+  depends_on "flex"
```

Comment:
```suggestion
  depends_on "bison" => :build
  depends_on "flex" => :build
```
Since these aren't used at runtime.

Reply:
Also, just to confirm, these don't work with the macOS versions of `bison` and `flex` right? I vaguely recall from #75486.

Reply:
Hmmm, I tested this locally without using Homebrew `flex` as a dependency and it builds and tests just fine.